deploy/helm/kubecf/values.yaml

```
@@ -212,6 +212,27 @@ services:
       start: 20000
       end: 20008
 
+settings:
+  router:
+    # tls sets up the public TLS for the router. The tls keys:
+    #   ca: the CA in the PEM format. Only required when the certificate is self-signed.
```

```suggestion
    #   ca: the CA in the PEM format. Not required if the certificate is signed by a well-known authority.
```
… Because technically a cert signed by the SUSE internal CA would _not_ be self-signed, but we would still require to give the CA cert here.